bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/thermostat/ThermostatHandler.java

```diff
@@ -1,10 +1,10 @@
 package org.openhab.binding.boschshc.internal.thermostat;
```

Comment:
This ``ThermostatHandler`` is similar to the new ``ClimateControlHandler`` device, therefore
- Does it make sense to move the ``ThermostatHandler`` class into a ``org.openhab.binding.boschshc.internal.devices...`` package?
- Can we use the new ``super.createService(...)`` function from ``BoschSHCHandler``?
- And add a class documenation?

Reply:
It's also fine for me if you like to finish the feature and refactor the code later

Reply:
Yes, that was the plan to do a big refactoring after our current pull requests were merged. So we put all services and devices into separate sub-packages (see https://github.com/stefan-kaestle/openhab2-addons/issues/7).

Reply:
Perfect! I got it and can & will support the big refactoring.

Reply:
Class looks now nice and clean 🎭